app/controllers/org_admin/sections_controller.rb

```diff
@@ -46,23 +46,20 @@ def edit
 
     # POST /org_admin/templates/[:template_id]/phases/[:phase_id]/sections
     def create
-      phase = Phase.includes(:template, :sections).find(params[:phase_id])
-      section = Section.new(section_params.merge({ phase_id: phase.id }))
+      section = Section.new(section_params.merge({ phase_id: params[:phase_id] }))
```

Comment:
this could create a data integrity issue since you are removing the phase find which automatically raises an ActiveRecord::RecordNotFound when an invalid phase id is passed...

Reply:
Yes good catch. Added a check in for the phase existence